x-pack/plugins/fleet/server/constants/fleet_es_assets.ts

```diff
@@ -9,6 +9,8 @@ import { getESAssetMetadata } from '../services/epm/elasticsearch/meta';
 
 const meta = getESAssetMetadata();
 
+export const FLEET_INSTALL_FORMAT_VERSION = '1.0.0';
```

Comment:
should we document somewhere (in README?) when the version should be incremented?